### PR TITLE
Added Squirrel helpers to generate Squirrel installers

### PR DESCRIPTION
**Pull request by andersosthus**
Added SquirrelHelpers.fs to generate Squirrel for Windows installers (https://github.com/Squirrel/Squirrel.Windows).

The code is a cleanup of a helper I made for internal use to create Squirrel installers.

I haven't done much F# before, so if you have suggestions to improvements, that would be greatly appreciated.

### CODE REVIEW

File: src/app/FakeLib/SquirrelHelper.fs [hunk 1]
```diff
@@ -0,0 +1,127 @@
+[<AutoOpen>]
```
Comment (forki) [inline src/app/FakeLib/SquirrelHelper.fs]: Please remove auto open

**Comment by forki** [comment]
This looks very cool. Need to take a deeper look.

File: src/app/FakeLib/SquirrelHelper.fs [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+/// - `SigningSecret` - `None`
+let SquirrelDefaults =
+    let toolname = "Squirrel.exe"
+    {
+        ReleaseDir = ""
+        WorkingDir = None
+        BootstrapperExe = None
+        LoadingGif = None
+        ToolPath = findToolInSubPath toolname (currentDirectory @@ "tools" @@ "Squirrel")
+        TimeOut = TimeSpan.FromMinutes 5.
```
Comment (paulcbetts) [inline src/app/FakeLib/SquirrelHelper.fs]: This should probably be 10min, this can take awhile

**Comment by paulcbetts** [comment]
LGTM!

**Comment by andersosthus** [comment]
Yeah, I've noticed that it can take some time, so a good idea to have the default higher.

**Comment by forki** [comment]
I'm on vacation for a couple of days, but I will merge/release  when I'm back on Monday.

**Comment by forki** [comment]
police

**Comment by sergey-tihon** [comment]
OMG OMG OMG
I definitely should update our build scripts
@andersosthus Thank you!

**Pull request merged**